lib/chartUtils.ts

```
@@ -0,0 +1,93 @@
+  }
+
+  const domainMax = Math.max(maxTick, maxValue);
+
+  return {
+    yDomain: [0, domainMax],
+    yTickValues: ticks,
+  };
+}
+
+export function formatYAxisLabel(value: number | string): string {
+  const numY = typeof value === 'number' ? value : parseFloat(value?.toString() || '0');
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)